pkg/crc/machine/machine.go

```diff
@@ -128,14 +128,16 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error getting bundle metadata: %v", err)
 		}
 
-		// Check if certificate is going to expire in next 7 days
-		buildTime, err := crcBundleMetadata.GetBundleBuildTime()
+		// Check if certificate is expired/going to expire in the next 7 days
+		err := checkBundleCertsExpiry(crcBundleMetadata)
```

Comment:
`s/Expire/Expiration` ?

Reply:
Probably https://writingexplained.org/expiry-vs-expiration-difference

Reply:
git master is already using Expiry rather than Expiration. I'll add a commit to the end of the series changing Expiry to Expiration, we can the discuss whether to keep it or not :)

Reply:
I added a commit on top of the PR doing the change, but the codebase is currently (before this PR) using expiry, so I'm not sure it's worth changing.